src/main/java/net/mrscauthd/beyond_earth/events/Methods.java

```diff
@@ -565,20 +567,17 @@ public static void setModelProperties(AbstractClientPlayer p_117819_, PlayerMode
         } else {
             if (p_117795_.getUsedItemHand() == p_117796_ && p_117795_.getUseItemRemainingTicks() > 0) {
                 UseAnim useanim = itemstack.getUseAnimation();
-                if (useanim == UseAnim.BLOCK) {
-                    return HumanoidModel.ArmPose.BLOCK;
-                }
-                else if (useanim == UseAnim.BOW) {
-                    return HumanoidModel.ArmPose.BOW_AND_ARROW;
-                }
-                else if (useanim == UseAnim.SPEAR) {
-                    return HumanoidModel.ArmPose.THROW_SPEAR;
-                }
-                else if (useanim == UseAnim.CROSSBOW && p_117796_ == p_117795_.getUsedItemHand()) {
-                    return HumanoidModel.ArmPose.CROSSBOW_CHARGE;
-                }
-                else if (useanim == UseAnim.SPYGLASS) {
-                    return HumanoidModel.ArmPose.SPYGLASS;
+                switch(useanim) {
+                    case BLOCK: return HumanoidModel.ArmPose.BLOCK;
+                    case BOW: return HumanoidModel.ArmPose.BOW_AND_ARROW;
+                    case SPEAR: return HumanoidModel.ArmPose.THROW_SPEAR;
+                    case CROSSBOW: {
+                        if (p_117796_ == p_117795_.getUsedItemHand()) {
+                            return HumanoidModel.ArmPose.CROSSBOW_CHARGE;
+                        }
+                        break;
+                    }
+                    case SPYGLASS: return HumanoidModel.ArmPose.SPYGLASS;
```

Comment:
`switch` is faster than a long chain of `else if` and is more concise.

The `break` inside CROSSBOW is used to tell it not to carry on with the other checks if it's a crossbow. We don't need to put break in the other cases as it returns.